pulp_2to3_migration/tests/functional/test_file_migration.py

```diff
@@ -18,36 +19,55 @@
 )
 from pulp_2to3_migration.pulp2.base import Repository as Pulp2Repository, RepositoryContentUnit
 from pulp_2to3_migration.pulp2.connection import initialize
+from pulp_2to3_migration.tests.functional.util import monitor_task
+
 
 # Can't import ISO model due to PLUGIN_MIGRATORS needing Django app
 # from pulp_2to3_migration.app.plugin.iso.pulp2_models import ISO
 
 
-def monitor_task(tasks_api, task_href):
-    """Polls the Task API until the task is in a completed state.
+# Initialize MongoDB connection
+initialize()
+
 
-    Prints the task details and a success or failure message. Exits on failure.
+PULP_2_REPOSITORIES = Pulp2Repository.objects.all()
 
-    Args:
-        tasks_api (pulpcore.client.pulpcore.TasksApi): an instance of a configured TasksApi client
-        task_href(str): The href of the task to monitor
+PULP_2_ISO_FIXTURE_DATA = {
+    # {'file': 3, ...}
+    repo.repo_id: RepositoryContentUnit.objects.filter(repo_id=repo.repo_id).count()
```

Comment:
since we also have docker repos populated, i think we should filter pulp2_repos by iso type, to exclude docker type.